frontend/Themes/Editor/ColorInput.js

```diff
@@ -77,6 +77,8 @@ class ColorInput extends React.Component {
 
     const backgroundIsBright = isBright(theme.base00);
     const chipIsBright = isBright(color);
+    const showColorChipBorder = backgroundIsBright === chipIsBright &&
+      getBrightness(color) > getBrightness(theme.base03);
```

Comment:
This change wasn't really related to the overall PR but I noticed it while testing the hover colors and thought I'd fix it while I was in the file.